packages/frontend/src/app/utils/index.ts

```diff
@@ -56,7 +56,34 @@ export const GetPostSummaries = (posts: any[]): PostSummary[] => {
       category: subSubcategory?.subcategory?.name,
       subSubcategory: subSubcategory?.name,
       publishedDate: post.publishedDate,
-      theme: GetThemeFromCategory(subSubcategory?.subcategory?.category?.slug),
+      theme: getThemeFromCategory(subSubcategory?.subcategory?.category?.slug),
     }
   })
 }
+
+export enum LogLevel {
+  INFO = 'INFO',
+  WARNING = 'WARNING',
+  ERROR = 'ERROR',
+}
+
+export const log = (level: LogLevel = LogLevel.INFO, msg: string) => {
+  const structuredMsg = JSON.stringify({
+    severity: level,
+    timestamp: new Date().toString(),
```

Comment:
Question:
Could we skip printing `timestamp` since Cloud Run will automatically add `timestamp` field for every log?

Reply:
removed.